testCourse/questions/addNumbersParameterized/server.py

```diff
@@ -0,0 +1,21 @@
+import random
+
+
+def generate(data):
+    lower_bound = data["params"].get("lower_bound", 10)
+    upper_bound = data["params"].get("upper_bound", 15)
```

Comment:
From previous discussion, we want to have these under a different key: https://github.com/PrairieLearn/PrairieLearn/pull/3848#discussion_r884013027